src/content/contributing/translation-program/playbook/index.md

```diff
@@ -0,0 +1,313 @@
+- *How are you planning on communicating with the translators?*
+    - If you are planning on using a localization tool, the communication can take place directly in the tool. Setting up an alternative communication channel with the translators is also recommended, since they might be less hesitant to reach out and messaging tools allow for more free-flowing communication.
+- *How to handle questions from translators? Who should be answering these questions?*
+    - Translators (both professional and non-professional) will often reach out with questions and requests for clarification or additional context, as well as feedback and ideas for improvements. Replying to these inquiries can often lead to better engagement and quality of translated content. It is also valuable to provide them with as many resources as possible (e.g., guides, tips, terminology guidelines, FAQs, etc.).
+- *How to handle the review process? Do you want to outsource it, or do you have the capacity to perform reviews internally?*
+    - While not always necessary, reviews are an integral part of an optimal translation process. Usually, it is easiest to outsource the review process to professional reviewers. However, if you have a large international team, the reviews or QA can also be handled internally.
+    
+4. **Implementing the translated content** – The last part of the workflow, though still important to consider ahead of time.
+- *Will all the translations be completed at the same time?*
+    - If not, you should think about which translations should be prioritized, how to keep track of the translations in progress, and how the implementation is handled while the translations are done.
+- *How will the translated content be delivered to you? What format will it be in?*
+    - This is an important consideration, regardless of which approach you use. Localization tools allow you to maintain control over the target file format and export process, and usually support automation, e.g., by enabling integration with the hosting service.
+- *How will you be implementing the translations in your project?*
+    - In some cases, this could be as simple as uploading the translated file or adding it to your docs. However, with more complex projects, like website or app translations, you should make sure the code supports internationalization, and establish how the implementation process will be handled ahead of time.
+- *What happens if the formatting is different to the source?*
+    - Similar to the above, if you are translating simple text files, the formatting probably isn’t crucially important. However, with more complex files, like content for a website or application, the formatting and code need to be identical to the source in order to be implemented in your project. If not, the target files will need to be edited, either by the translators or your developers.
```

Comment:
```suggestion
1. **Preparing the files for translation** – It sounds simple; however, you need to consider a couple of important things. At this step, you should have a clear plan on how the entire process should work. 
- *Which file types will you be using? What format do you want to receive your translated files in?*
    - If your content is available in DOCX or MD format, the approach will be much more straightforward than if you are translating a PDF version of your Whitepaper or other documents.
- *Which localization tools support this file type? Can the file be translated in a way that retains the original formatting?*
    - Not all file types support direct localization (e.g., PDF files, image files), and not all localization tools support all file types.
- *Who will be translating the content? Will you be ordering professional translations or relying on volunteers?*
    - This affects a number of other decisions you need to make. For example, professional translators are more comfortable working with advanced localization tools than volunteers.
- *What are your expectations for the linguists? If you are using a language service provider, what do they expect from you?*
    - This is the step to make sure your goals, expectations, and timelines are aligned.
- *Is all the content for translation equally important? Should some content be translated first?*
    - There are some ways to prioritize certain content, which should be translated and implemented first. For example, if you have a lot of content for translation, you can use version control to make sure the translators are aware of which they should prioritize.

2. **Sharing the files for translation** – This step also requires some long-term thinking and is not as straightforward as sending the source files to a language service provider.
- *Who will be translating the content? How many people will be involved in this process?*
    - If you plan to use a localization tool, this step is simplified since you can upload the source files to the tool directly. This is also true if the translation process takes place on the hosting service since the source files don’t need to be exported anywhere.
- *Will the source files be handled manually, or can this process be automated?*
    - Most localization tools allow for some type of integration or automation of the file management process. On the other hand, if you are working with individual translators and not using a localization tool, manually sending source files to hundreds or thousands of translators is not a scalable process.
- *Which tools will be used for the localization?*
    - The answer to this question will determine how you approach everything else. Selecting the proper tool can help you automate content management, managing the Translation Memory and Glossary, managing translators, keeping track of the translation/review progress, etc., so take some time and do some research on which tool you want to use. If you are not planning on using a localization tool, all of the above will need to be done manually.
- *How long will the translation process take? How much will it cost?*
    - At this point, you should be ready to share the source files with the language service provider or pool of translators. The language service provider can help you analyze the word count and provide a quote, including the rates and timeline for the translation process.
- *Are you planning on making changes/updating the source content during this process?*
    - If your content is dynamic and changes often, any changes or updates can disrupt the translation progress. Using a Translation Memory can help mitigate this significantly, though it is still important to think about how the process will work and how you can prevent setting back the progress the translators are making.

3. **Managing the translation process** – Your work isn’t done once the source content is handed off to the language service provider or the translators. To ensure optimal quality of the translations, content creators should be as involved with the translation process as possible.
- *How are you planning on communicating with the translators?*
    - If you are planning on using a localization tool, the communication can take place directly in the tool. Setting up an alternative communication channel with the translators is also recommended since they might be less hesitant to reach out, and messaging tools allow for more free-flowing communication.
- *How to handle questions from translators? Who should be answering these questions?*
    - Translators (both professional and non-professional) will often reach out with questions and requests for clarification or additional context, as well as feedback and ideas for improvements. Replying to these inquiries can often lead to better engagement and quality of translated content. It is also valuable to provide them with as many resources as possible (e.g., guides, tips, terminology guidelines, FAQs, etc.).
- *How to handle the review process? Do you want to outsource it, or do you have the capacity to perform reviews internally?*
    - While not always necessary, reviews are an integral part of an optimal translation process. Usually, it is easiest to outsource the review process to professional reviewers. However, if you have a large international team, the reviews or QA can also be handled internally.

4. **Implementing the translated content** – The last part of the workflow, though still important to consider ahead of time.
- *Will all the translations be completed at the same time?*
    - If not, you should think about which translations should be prioritized, how to keep track of the translations in progress, and how the implementation is handled while the translations are done.
- *How will the translated content be delivered to you? What format will it be in?*
    - This is an important consideration, regardless of which approach you use. Localization tools allow you to maintain control over the target file format and export process and usually support automation, e.g., by enabling integration with the hosting service.
- *How will you be implementing the translations in your project?*
    - In some cases, this could be as simple as uploading the translated file or adding it to your docs. However, with more complex projects, like website or app translations, you should make sure the code supports internationalization and establish how the implementation process will be handled ahead of time.
- *What happens if the formatting is different to the source?*
    - Similar to the above, if you are translating simple text files, the formatting probably isn’t crucially important. However, with more complex files, like content for a website or application, the formatting and code need to be identical to the source in order to be implemented in your project. If not, the target files will need to be edited, either by the translators or your developers.
```